src/Event/Sns/MessageAttribute.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types=1);
+
+namespace Bref\Event\Sns;
+
+/**
+ * SNS message attribute.
+ *
+ * @see https://docs.aws.amazon.com/sns/latest/api/API_MessageAttributeValue.html
+ * @see https://github.com/aws/aws-lambda-java-libs/blob/master/aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/SNSEvent.java
+ */
+class MessageAttribute
+{
+    /** @var array */
+    private $attribute;
+
+    public function __construct(array $attribute)
+    {
+        $this->attribute = $attribute;
```

Comment:
May it be necessary to check that there is an Index Type and Value?

Reply:
I'm not sure, at this point the shape of the event should have been validated earlier. I don't validate all keys in all value objects.

Reply:
ok, usually I validate all keys to ensure the correct use of the value object. But it's not a block for this PR I think